test/tests-index_notation.cpp

```diff
@@ -153,6 +153,30 @@ TEST(notation, isomorphic) {
   ASSERT_FALSE(isomorphic(sum(j, B(i,j) + C(i,j)), sum(j, B(j,i) + C(j,i))));
 }
 
+TEST(notation, generatePackCOOStmt) {
+  TensorVar ac("a_COO", vectorType);
+  ASSERT_NOTATION_EQ(forall(i, a(i) = ac(i)), 
```

Comment:
Can instead use the `isomorphic` function to validate `generatePackCOOStmt` if its signature is changed (as per above).

Reply:
Changed to use isomorphic.